weareinreach/inreach-catalog
# user can delete a favorites list

## Description

From the favorites list, a user can select a list, once selected, they now also have the option to delete it. 

language files have been updated for all text, the Delete button, the delete confirmation message

button text is also generated from the language files and value is passed as a prop to the button component

other tweaks:
- AsylumConnectButton - add some margin to the base button to remove overlap when there are multiple buttons in the dialog
- DeleteAccountDialog - change the Cancel button to secondary to better distinguish it from the Delete button. - Delete List Dialog and Delete Account Dialog will have a similar style.

## Asana ticket:
https://app.asana.com/0/1200689352502326/1200865997295677

## PR Checklist




## How to Test

1. User clicks the Delete button, a dialog opens which will have a confirmation message, the list title and two buttons. If the list had previously been shared an additional message will appear. along the lines of, this list has been shared.....
2. The user can choose Delete or Cancel. 
3. If they choose 'Cancel' the dialog will close, and the user will remain on the view of the  selected list.
4. If they choose "Delete", the list will be deleted from the DB. the dialog will close, once closed the user will return the the Favorites list Home view - The list which had been deleted should no longer appear in the Favorites list.

should validate in Desktop as well as mobile view
